src/Framework/Str.php

```diff
@@ -51,51 +53,116 @@ final class Str
 {
     use CannotBeInstantiated;
 
-    public static function trimLineReturns(string $string): string
+    private const LINE_RETURNS_REGEX = '/\r\n|\r|\n/';
+
+    /**
+     * @psalm-suppress InvalidReturnStatement,InvalidReturnType
+     */
+    public static function toSystemLineReturn(string $value): string
+    {
+        return preg_replace(
```

Comment:
note here that I used `preg_replace()` because I found it simpler at that time, but it is also managable with `strtr`:

```php
return strtr(
            $string,
            [
                "\r\n" => "\n",
                "\r"   => "\n",
            ],
        );
```

in case you have a preference...